performance/scripts/preScript.js

```diff
@@ -1,18 +1,29 @@
 const http = require('http');
 
-module.exports = async function (context) {
+module.exports = async function (context, commands) {
 
     const hostname = 'demo';
     const port = '8080';
+    const path = 'performance/performance.html';
 
     return new Promise(resolve => {
         function poll() {
-            setTimeout(() => {
-                http.get(`http://${hostname}:${port}`, (res) => {
+            setTimeout(async () => {
+                http.get(`http://${hostname}:${port}`, async (res) => {
                     const { statusCode } = res;
                     if (statusCode === 200) {
-                        context.log.info(`App available on ${hostname}:${port}.`);
-                        resolve();
+                        try {
+                            context.log.info(`App available on ${hostname}:${port}.`);
+                            await commands.navigate(`http://${hostname}:${port}/${path}`);
+                            const demoComponents = await commands.js.run('return document.querySelectorAll(".demo").length > 0');
```

Comment:
volgens mij niet afdoende om te verzekeren dat er wel degelijk "iets" gerendered is